tests/cases/fourslash/fourslash.ts

```diff
@@ -70,6 +70,7 @@ declare module ts {
         Type = "Type",
         Parameter = "Parameter",
         Enum = "Enum",
+        Other = "Parenthesis"

```

Comment:
This needs to be updated I assume - but how are tests passing?

Reply:
Fourslash does not affect piblic api baseline.

Reply:
Sorry, updated.